docs/dapps/build-dapp-with-nextjs.md

```diff
@@ -128,20 +128,17 @@ After user is connected to the wallet, we can interact with the
 Alephium blockchain by using a set of react hooks provided by
 [@alephium/web3-react](https://github.com/alephium/alephium-web3/tree/master/packages/web3-react). For
 example, getting the [current
-account](https://github.com/alephium/alephium-web3/blob/master/packages/web3-react/src/hooks/useAccount.tsx),
+connected wallet](https://github.com/alephium/alephium-web3/blob/master/packages/web3-react/src/hooks/useWallet.tsx),
 [balance](https://github.com/alephium/alephium-web3/blob/master/packages/web3-react/src/hooks/useBalance.tsx)
 and [transaction
 status](https://github.com/alephium/alephium-web3/blob/master/packages/web3-react/src/hooks/useTxStatus.tsx),
 etc.
 
-It is worth mentioning that the `useBalance` hook also provides the functionality to update the balance.
+When a user makes a transaction, you can update the user's balance using `updateBalanceForTx`.
 Here is a simple example:
 
 ```typescript
-// The useBalance hook returns two values:
-// 1. balance: the current balance
-// 2. updateBalanceForTx: used to update the balance when the user makes a transaction.
-const { balance, updateBalanceForTx } = useBalance()
+const { updateBalanceForTx } = useAlephiumBalanceContext()
```

Comment:
Should we use `useBalance` instead of using context here? Otherwise, the first balance query will not be triggered

Reply:
Just found that `useBalance` only returns balance 😓, do we need to fix this in sdk?